### PR TITLE
Documentation/note overriding scaffolding phases

### PR DESCRIPTION
**Pull request by onlyhavecans**
I couldn't find any documentation commenting on this fact and it hung me up for about an hour so I thought it was worth putting a note in the documentation.

If someone has any better phrasing I'm open to it.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/partials/docs/_glossary-scaffolding.html.md.erb [hunk 1]
```diff
@@ -14,6 +14,10 @@ While we are targeting many platforms for automated scaffolding we currently sup
 
 Scaffoldings provide certain overrideable variables for language-specific behavior.  Please see the appropriate scaffolding documentation for details.
 
+## Overriding scaffolding callbacks
```
Comment (cnunciato) [inline www/source/partials/docs/_glossary-scaffolding.html.md.erb]: Nit: I'd probably capitalize this heading just for consistency with the other sections.

**Comment by onlyhavecans** [comment]
Updated with the correction on caps 👍

**Comment by elliott-davis** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/335214164).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/335214164).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/335214164) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by onlyhavecans** [comment]
Is there a master breakage or intermittent failure. I looked at the failure in travis-ci and it seems to be code related, not related to the docs. Is someone can point me to what's causing the failure I can try to remit

**Comment by tashimi** [comment]
@onlyhavecans in my experience, when I hit this error it means I need to rebase

**Comment by onlyhavecans** [comment]
@tashimi I'm super unsure how such a minor docs change causes an issue but I rebased from origin master so hopefully this is all fixed.

**Comment by tashimi** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/335328307).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/335328307).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by cnunciato** [comment]
Thanks @onlyhavecans! 

@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/335389180).

**Comment by tashimi** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/335391253).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/335389180).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to pass the test suite](https://travis-ci.org/habitat-sh/habitat/builds/335391253).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/335391253) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Pull request merged**